docs/source/reference/job-queue.rst

```diff
@@ -1,24 +1,25 @@
 .. _job-queue:
 
-Cluster Job Queue
-=================
+Cluster Jobs
+=============
 
-SkyPilot's **job queue** allows multiple jobs to be scheduled on a cluster.
+You can run jobs on an existing cluster, which are automatically queued and scheduled.
+
+This is ideal for interactive development on an existing cluster and reusing its setup.
 
 Getting started
 --------------------------------
 
-Each task submitted by :code:`sky exec` is automatically queued and scheduled
-for execution on an existing cluster:
+Use :code:`sky exec` to submit jobs to an existing cluster:
```

Comment:
There are still a few `task` in `Multi-node Jobs`. Should we update them? Also, should we use `Cluster Jobs` there and also add a section mentioning `Managed Jobs` can directly work with multi-node as well?